babel.config.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
   ],
   'plugins': [
     'babel-plugin-transform-es2015-modules-commonjs',
```

Comment:
can this be removed now that you added `@babel/plugin-transform-modules-commonjs`?

Reply:
Agree, looks like unnecessary plugin.